mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -361,38 +360,36 @@ class ExperimentView extends Component {
     const numParams = paramKeyList.length;
     const numMetrics = metricKeyList.length;
     const row = [
-      <td><input type="checkbox" checked={selected}
+      <td key="Hcheck"><input type="checkbox" checked={selected}
         onClick={() => onCheckbox(runInfo.run_uuid)}/></td>,
-      <td>
+      <td key="Hlink">
         <Link to={Routes.getRunPageRoute(runInfo.experiment_id, runInfo.run_uuid)}>
           {runInfo.start_time ? Utils.formatTimestamp(runInfo.start_time) : '(unknown)'}
         </Link>
       </td>,
-      <td>{Utils.formatUser(runInfo.user_id)}</td>,
-      <td>{Utils.renderSource(runInfo)}</td>,
-      <td>{Utils.renderVersion(runInfo)}</td>,
+      <td key="Huser">{Utils.formatUser(runInfo.user_id)}</td>,
+      <td key="Hsource">{Utils.renderSource(runInfo)}</td>,
+      <td key="Hversion">{Utils.renderVersion(runInfo)}</td>,
     ];
 
-    let firstParam = true;
-    paramKeyList.forEach((paramKey) => {
-      const className = firstParam ? "left-border": undefined;
-      firstParam = false;
+    paramKeyList.forEach((paramKey, i) => {
+      const className = i === 0 ? "left-border": undefined;
+      const keyname = "0" + paramKey;
       if (paramsMap[paramKey]) {
-        row.push(<td className={className}>
+        row.push(<td className={className} key={keyname}>
           {paramsMap[paramKey].getValue()}
         </td>);
       } else {
-        row.push(<td className={className}/>);
+        row.push(<td className={className} key={keyname}/>);
       }
     });
     if (numParams === 0) {
-      row.push(<td className="left-border"/>);
+      row.push(<td className="left-border" key={"HParamEmpty"}/>);
     }
 
-    let firstMetric = true;
-    metricKeyList.forEach((metricKey) => {
-      const className = firstMetric ? "left-border": undefined;
-      firstMetric = false;
+    metricKeyList.forEach((metricKey, i) => {
+      const className = i === 0 ? "left-border": undefined;
+      const keyname = "1" + metricKey;
```

Comment:
Call it `"metric-" + paramKey` to make it easier to debug. Same with the other places it shows up.